### DISCUSSION TITLE
Tree View

### DISCUSSION THREAD

**Discussion by mule-stand** [ariakit/ariakit#4311] [Questions] [answered]
Is there any possibility to implement a Tree UI?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tree_role

The request is that I want to realize a similarity of this.
![image](https://github.com/user-attachments/assets/cab4b1b3-c0ab-4660-9a2a-90a662fce4a2)

Examples:
https://mui.com/x/react-tree-view/

**Comment by ciampo** [comment, marked as answer]
This is being discussed in https://github.com/ariakit/ariakit/issues/4059, where @diegohaz said that there are plans, but currently no ETA.
